hack/update-gofmt.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Sorry, why were this and the `lib/go.sh` files added ?

Reply:
ah, ok, this one requires the lib/go.sh then why was this one added ?

Reply:
My IDE does not always formats the code properly. Running ./hack/update-gofmt is a very quick way to do it.